examples/fishjam-chat/screens/RoomScreen.tsx

```diff
@@ -47,40 +47,27 @@ const RoomScreen = ({ navigation, route }: Props) => {
 
   const tracks = useMemo(() => parsePeersToTracks(peers), [peers]);
 
-  const { toggleScreenShare, isScreenShareOn, handleScreenSharePermission } =
-    useScreenShare();
+  const { toggleScreenShare, isScreenShareOn } = useScreenShare();
 
   const onDisconnectPress = useCallback(() => {
     leaveRoom();
     navigation.navigate('Home');
   }, [navigation]);
 
   useForegroundService({
+    channelId: 'io.fishjam.example.fishjamchat.foregroundservice.channel',
+    channelName: 'Fishjam Chat Notifications',
+    notificationTitle: 'Your video call is ongoing',
+    notificationContent: 'Tap to return to the call.',
     enableCamera: isCameraOn,
     enableMicrophone: isMicrophoneOn,
   });
 
-  const handleAndroidScreenSharePermission = useCallback(
-    async (isScreenShareOn: boolean) => {
-      if (isScreenShareOn) {
-        return;
-      }
-      if ((await handleScreenSharePermission()) != 'granted') {
-        return;
-      }
-    },
-    [handleScreenSharePermission],
-  );
-
   const onToggleScreenShare = useCallback(async () => {
```

Comment:
**nitpick:** 
Name of the function suggest that it is called when screenShare is toggled, meanwhile it is calling toggleScreenShare. I think name of this function should be changed to  `toggleScreenShare`.

Reply:
Let it be. I see that this code is not even your so you can leave it as it is.